lib/ace/mode/xml/sax.js

```diff
@@ -448,10 +451,17 @@ function fixSelfClosed(source,elStartEnd,tagName,closeMap){
 	var pos = closeMap[tagName];
 	if(pos == null){
 		//console.log(tagName)
-		pos = closeMap[tagName] = source.lastIndexOf('</'+tagName+'>')
+		var regex = new RegExp("</"+tagName+"\\s*>", "gi");
+		var match      = source.match(regex);
+		if (match && match.length >= 1) {
+			var lastIndex  = source.lastIndexOf(match[match.length-1]);
+			pos = closeMap[tagName] = lastIndex;
+		} else {
+			pos = -1;
```

Comment:
Can we also keep the same chain assignment from above here, so `pos = closeMap[tagName] = -1` just to keep the  previous logic the same.